src/app/design/guidelines/page-layouts/index.html

```diff
@@ -90,6 +90,17 @@
     The following sections describe the standard SKY UX page layouts. These layouts show how to package components to suit a variety of user needs, but the system can accommodate alternate layouts as necessary.
   </p>
 
+  <div class="sky-subsection-heading" style="margin-top: 20px;">
+    Sectioned page
+  </div>
+
+  <p>
+    A sectioned page is a simple layout that defines page sections with typography and spacing. Use vertical page sections to categorize the relevant content on the page and group. Use <a routerLink="/components/fluid-grid">fluid grid</a> to arrange content horizontally. The page can use most content containers like page summary, tabs, repeaters, cards, etc. Keep the number of sections low (5 or fewer), or consider breaking the content up with the additional tabs or the tile-based Work center layout.
```

Comment:
I don't understand what the second sentence means. What does "to categorize the relevant content on the page and group" mean? And why does it only apply to vertical sections? Also, the content container topic makes it sound like you use typography and spacing for vertical page sections OR you use fluid grid for horizontal page sections. But this first sentence makes it sounds like you use typography and spacing for all page sections, which suggests that you use typography, spacing, AND fluid grid for horizontal sections. Which is correct? ... I suspect we should change these opening sentences to something like "The sectioned page layout provides a simple way to organize page contents into categories and groups. You can divide pages into vertical sections with typography and spacing, and you can divide pages into horizontal sections with the fluid grid component." But I'm not 100 percent sure if that's an accurate assessment of how this works.

In the third sentence, I'd change "The page" to "Sectioned pages." I'd also change "like" to "such as" and delete "etc." and add "and" before "cards."

In the last sentence, I'd change the opening clause to "Limit the number of sections to 5 or fewer." And I'd make the second half of the sentence a new sentence that makes it clearer that if you need more than 5 sections, you should try something different: "If you need more than 5 sections, break content up tabs or the tile-based work center layout."